packages/transition-backend/src/tasks/TransitionWorkerPool.ts

```diff
@@ -65,7 +71,7 @@ const getTaskCancelledFct = (task: ExecutableJob<JobDataType>) => {
                     clearInterval(intervalObj);
                 }
             })
-            .catch(() => (refreshError = true));
+            .catch(() => (refreshError = true)); // This will catch deleted jobs
```

Comment:
Just checking, you added the same comment as before without changing the code. This is accurate and not just a bad copy/paste ?

Reply:
It's accurate, it adds an explanation of which kind of use case would be caught here.